{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "4af42ddf_bd049d9b",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 11
      },
      "lineNbr": 0,
      "author": {
        "id": 1000938
      },
      "writtenOn": "2025-09-12T05:58:45Z",
      "side": 1,
      "message": "Can you try a rebase and solve the CI failure?",
      "revId": "a135662143020749f37a2af60d7b9c506cb13b5f",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}